eclipse-theia/theia
# Align quickpick labels

#### What it does

This is a very minor change. In the quickpick menu when searching for a file, some results have a generic icon image. For those results, the path is vertically offset.

![image](https://user-images.githubusercontent.com/20653241/222216906-9bbc9eea-a7d5-4cf3-8075-2dff4b2e7e5f.png)

With the change:

![image](https://user-images.githubusercontent.com/20653241/222217193-76f932c6-ce58-4f83-94a7-0108428b343c.png)

A more perfect change would refactor the CSS to avoid the unnecessary hard-coded pixel values in `monaco-editor-core` that lead to this. However, that would also have complex effects. The advantage of this PR's change is that it's only within a highly specific CSS selector, so should not interfere anywhere else.

#### How to test


1. Open the quickpick to search for a file (`Ctrl + P`). Search for various strings and observe the icons in the list of results. For only the generic file icon, Theia on `main` has misaligned path labels.
2. On this branch, the path labels should be correctly aligned.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
